tests/trestlebot/tasks/authored/test_ssp.py

```diff
@@ -246,28 +246,56 @@ def test_create_new_with_filter(tmp_trestle_dir: str) -> None:
     _ = testutils.setup_for_profile(trestle_root, test_prof_filter, test_prof_filter)
 
     ssp_name = "new_ssp"
```

Comment:
Consider using os.path.join a comment explaining new directory to be created with ssp name